src/lib/Components/Bidding/Screens/RegistrationResult.tsx

```diff
@@ -59,6 +65,9 @@ export class RegistrationResult extends React.Component<RegistrationResultProps,
         title = registrationPendingMessage.title
         msg = registrationPendingMessage.description
         break
+      case RegistrationStatus.RegistrationStatusNetworkError:
+        title = registrationNetworkErrorMessage.title
+        msg = registrationNetworkErrorMessage.description
```

Comment:
This is missing a case for the `"RegistrationStatusError"` key and the result screen may end up displaying an empty title and description.

Reply:
That ones is covered in the `default` case